pkg/operator/ceph/object/notification/obc_label_controller.go

```diff
@@ -161,24 +165,32 @@ func (r *ReconcileOBCLabels) reconcile(request reconcile.Request) (reconcile.Res
 				return reconcile.Result{}, errors.Wrapf(err, "failed to retrieve CephBucketNotification %q", bnName)
 			}
 
+			r.recorder.Eventf(notification, kapiv1.EventTypeNormal, string(cephv1.ReconcileStarted), "Start reconciling %q", bnName)
```

Comment:
For integration tests, it will be important to know when reconcile begins due to OBC changes and when due to Notification changes.

```suggestion
			r.recorder.Eventf(notification, kapiv1.EventTypeNormal, string(cephv1.ReconcileStarted), "Started reconciling CephObjectBucketNotification %q for ObjectBucketClaim %q", bnName, obcNamespacedName)
```